sympy/solvers/ode/single.py

```diff
@@ -991,6 +991,78 @@ def _get_general_solution(self, *, simplify_flag: bool = True):
         return [gensol]
 
 
+class RationalRiccati(SinglePatternODESolver):
+    r"""
+    Gives all rational solutions to the first
+    order Riccati Differential Equation
```

Comment:
This doesn't just find rational solutions since e.g. `rational_riccati_15` has exponential functions.

Reply:
Yes. Maybe I can say that it finds general solutions for all Riccati equations that have atleast 1 particular rational solution?